core/src/main/scala/org/apache/spark/util/AccumulatorV2.scala

```diff
@@ -114,8 +114,8 @@ abstract class AccumulatorV2[IN, OUT] extends Serializable {
   final private[spark] def isAtDriverSide: Boolean = atDriverSide
 
   /**
-   * Returns if this accumulator is zero value or not. e.g. for a counter accumulator, 0 is zero
-   * value; for a list accumulator, Nil is zero value.
+   * Returns false if this accumulator has been updated. Note that this can be true even when
+   * the value is a zero value, if the accumulator was updated with a zero value.
```

Comment:
I think the previous comment is fine. Each accumulator can define the semantic of zero value by its own.

Reply:
That's fair, reverted